HISTORY.rst

```diff
@@ -4,8 +4,14 @@
 TaskGraph Release History
 =========================
 
-Unreleased Changes
+Unreleased changes
 ------------------
+* Fixed issue that would cause an infinate loop if a ``TaskGraph`` object were
+  created with a database from an incomptable previous version. Behavior now
+  is to log the issue, delete the old database, and create a new compatable
+  one.
+* Fixed issue that would cause some rare infinate loops if ``TaskGraph`` were

```

Comment:
just a typo: `infinite`

Reply:
Ha that wasn't the only typo. I was thinking why didn't spell check catch that and it's because it wasn't on. At least a dozen more are fixed in 8bfdeb2dbd721b688115b9c6ec3d136d86ee0829